starter/src/main/java/org/teiid/spring/autoconfigure/TeiidServer.java

```diff
@@ -281,6 +283,15 @@ String getDriverName(Object source) {
         return driverName;
     }
 
+    String buildNewTranslatorName(VDBMetaData vdb, String proposedName) {
```

Comment:
@shawkins now I added verification that source names are not reused as translator names, this will check for any duplications.